.travis.yml

```diff
@@ -161,7 +161,7 @@ script:
 
         (make check-unit-coverage \
          && do_codecov unittests \
-         && sed "1 s~^~require('luacov.runner')('$PWD/.luacov'); \0~" build/awesomerc.lua \
+         && sed -i "1 s~^~require('luacov.runner')('$PWD/.luacov'); \0~" build/awesomerc.lua \
```

Comment:
Ok.  I will change tests/run.sh to handle this (instead of here), which would allow for `DO_COVERAGE=1 make check` to handle it there.